src/main/java/lotto/Application.java

```diff
@@ -1,7 +1,21 @@
 package lotto;
 
+import lotto.controller.LottoController;
+import lotto.service.PurchaseService;
+import lotto.service.ResultService;
+import lotto.service.WinningNumberService;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
 public class Application {
+
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        OutputView outputView = new OutputView();
+        PurchaseService purchaseService = new PurchaseService();
+        ResultService resultService = new ResultService();
+        WinningNumberService winningNumberService = new WinningNumberService();
+        LottoController lottoController = new LottoController(outputView, purchaseService, resultService,
+                winningNumberService);
```

Comment:
```suggestion
        LottoController lottoController = new LottoController(
                        new OutputView(),
                        new PurchaseService(), 
                        new ResultService(),
                        new WinningNumberService);
```
으로 선언 해주는 건 어땠을까요? 그냥 저의 조그만... 의견입니다!

Reply:
최대한 의존성을 낮춰주고싶어서 저렇게 구현해보았는데 다양한 방법들을 공부해봐야겠네요:)